website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -51,8 +50,35 @@ Use color logically.
 - **Highlight** use interchangeably with `neutral` when more prominence is needed. Use it sparingly.
 - **Success** to indicate a successful action was completed.
 - **Warning** to indicate a successful action was completed but may have triggered a related issue. Provide guidance and actions if possible.
+
 - **Critical** to indicate error or critical issues resulting from a failed action.
 
+## Critical Toast vs Critical Alert
+
+### When to use a Toast
+
+Use the Toast to provide non-intrusive feedback to users about the failure of an ongoing task or request. For example, a failure while deleting a cluster.
+
+<Hds::Toast @color="critical" @onDismiss={{this.noop}} as |T|>
+    <T.Title>Module error</T.Title>
+    <T.Description>This module encountered an error during publishing. You may need to republish.</T.Description>
+  </Hds::Toast>
+
+Use the Toast to communicate error messages that are not caused by the user. For example, an unsuccessful Vault cluster creation due to a failure while validating the deployment.
+
+<Hds::Toast @color="critical" @onDismiss={{this.noop}} as |T|>
+    <T.Title>Vault cluster update failed</T.Title>
+    <T.Description>**test-cluster_1** configuration update failed.</T.Description>
+  </Hds::Toast>
+
+<!-- Do not set critical Toast to dismiss automatically. -->
+
+### When to use an Alert
```

Comment:
[suggestion] I wonder if showing the Alert examples (from the Alert Guidelines) inside a Toast wrapped in a don't block for each scenario would be more impactful. That would keep the documentation scoped to Toast guidance instead of bleeding Alert guidance into the Toast docs.

Reply:
+1

Reply:
I like this approach. I turned the section into a do/don’t usage section and adjusted the content.